### ISSUE TITLE
SQL Server Agent - Edit Job does not show steps

### ISSUE THREAD

**Issue by wsmelton** [microsoft/azuredatastudio#15924]
- Azure Data Studio Version:
```
Version: 1.31.0-insider (system setup)
Commit: 3eef3f2a0bfef1f6c5a852a575568a13ea50dc12
Date: 2021-06-25T05:35:39.155Z
VS Code: 1.56.0
Electron: 12.0.4
Chrome: 89.0.4389.114
Node.js: 14.16.0
V8: 8.9.255.24-electron.0
OS: Windows_NT x64 10.0.19042
````
- OS Version: Windows 10 20H2

Steps to Reproduce:

1. Download and run the [MaintenanceSolution.sql](https://ola.hallengren.com/scripts/MaintenanceSolution.sql) from Ola's site
2. That script will create a number of jobs with steps on them.
3. Edit the CommandLog cleanup job
4. Click on Steps, nothing is shown

![image](https://user-images.githubusercontent.com/11204251/123532313-bc43c900-d6d1-11eb-98ff-e72bdedb4461.png)

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by lewis-sanchez** [comment]
@wsmelton I followed your steps to reproduce the bug using both the insiders and General Availability (GA) versions of Azure Data Studio, and I was able to see the steps for the CommandLog Cleanup job. I didn't need to disable any of my extensions to get the job steps to appear when editing a job. Are the job steps missing when you edit any of the jobs or only for some of them? Does disabling extensions allow you to see the steps?

![image](https://user-images.githubusercontent.com/87730006/129653412-85989bf8-abb8-4059-b8b5-06f7cade8825.png)
